### ISSUE TITLE
Remove output of entire file that contains JSON schema combiner keyword

### ISSUE THREAD

**Issue by DerkSchooltink** [codekie/openapi-examples-validator#136]
I think the very obnoxious warning that contains the entire JSON should be removed. It fills up the entire log output and is not of very much help. 

Example:

```
Validating examples
"additionalProperties" flag not set for $['paths']['<redacted>']['get']['responses'][400]['content']['application/json']['schema']['oneOf'][0] because it contains JSON-schema combiner keywords.
"additionalProperties" flag not set for $['paths']['<redacted>']['get']['responses'][400]['content']['application/json']['schema']['oneOf'][1] because it contains JSON-schema combiner keywords.
"additionalProperties" flag not set for $['paths']['<redacted>']['get']['responses'][400]['content']['application/json']['schema']['oneOf'][0]['properties']['errors']['items'] because it contains JSON-schema combiner keywords.
"additionalProperties" flag not set for $['paths']['<redacted>']['get']['responses'][401]['content']['application/json']['schema']['oneOf'][0] because it contains JSON-schema combiner keywords.
"additionalProperties" flag not set for $['paths']['<redacted>']['get']['responses'][401]['content']['application/json']['schema']['oneOf'][1] because it contains JSON-schema combiner keywords.
"additionalProperties" flag not set for $['paths']['<redacted>']['get']['responses'][401]['content']['application/json']['schema']['oneOf'][1]['properties']['errors']['items'] because it contains JSON-schema combiner keywords.
"additionalProperties" flag not setfor {"oneOf":[<very long JSON content>]} because it contains JSON-schema combiner keywords.
Schemas with examples found: 10
Examples without schema found: 17
Total examples found: 44
```

There is also a small typo where `set` and `for` are linked together 😁 

What I propose is that not the schema is shown in the logs but the path to the JSON instead. 

The callback in the `disallow-additional-properties.js` would look something like this:

```
function _callbackObjectTypeForNoAdditionalProperties(path) {
    return (value) => {
        const asString = JSON.stringify(value);
        // any schema's that use the `allOf`-combiner should also be excluded
        if (!asString.includes(`"${COMBINER__ALL_OF}"`)) {
            value.additionalProperties = false;
        } else {
            console.warn('"additionalProperties" flag not set '
                + `for ${path} because it contains the "allOf" combiner keyword.`);
        }
    };
}
```

That way you can insert the callback like this:

```
for (const jsPath of paths) {
    _find(openApiSpec, jsPath, ResultType.value, callback(jsPath));
}
```

Let me know what you think @codekie 😄  If you agree, I can submit a PR.

**Comment by codekie** [comment]
I agree

**Comment by DerkSchooltink** [comment]
Fixed by #137
